unikernel.ml

```diff
@@ -233,9 +233,7 @@ struct
   let extract_json_csrf_token f token_or_cookie user reqd =
     extract_json_body reqd >>= function
     | Error (`Msg err) ->
-        Middleware.http_response reqd
-          ~data:(`String (String.escaped err))
-          `Bad_request
+        Middleware.http_response reqd ~data:(`String err) `Bad_request
```

Comment:
with the state of unikernel.ml after this commit, there are three String.escaped left -- and two of them are in exactly this case -- (here, extract_json_body calls `Yojson.Basic.of_string`, and the resulting `Json_error msg` -- should `msg` be escaped or not)? Since I don't understand yojson too well, I'm not sure...

but we should treat this uniform :)